proto/outbound.proto

```diff
@@ -75,14 +75,17 @@ message ProxyProtocol {
 
   message Http1 {
     repeated HttpRoute routes = 1;
+    Breaker breaker = 2;
   }
 
   message Http2 {
     repeated HttpRoute routes = 1;
+    Breaker breaker = 2;
   }
 
   message Grpc {
     repeated GrpcRoute routes = 1;
+    Breaker breaker = 2;
```

Comment:
in the proxy, this is currently referred to as a "failure accrual policy" although this naming is certainly open to change...

Reply:
also, should we maybe document that this field is allowed to be empty and that this indicates that no circuit breaking should be performed?

Reply:
I'm open to renaming this but `failure_accrual_policy` feels a bit verbose and overly prescriptive.  I could imagine other circuit breaker policies which trigger on conditions other than accrued failures (e.g. windowed success rate, latency, or other signals).  The term `breaker` felt concise and generic.

Reply:
i would consider a windowed success rate a form of failure accrual, but :woman_shrugging:. not a strongly held opinion.

Reply:
I think it's really important that we be crisp with our term. If these are breakers then we should stop referring to failure accrual everywhere else. Or the other way around.

In my view, circuit breakers are an emergent behavior that build off of a failure accrual configuration. I could go deeper on the distinction but for the sake of brevity, I think it's best to call this `FailureAccrual`